mono/metadata/assembly.c

```diff
@@ -133,36 +135,140 @@ static const AssemblyVersionMap framework_assemblies [] = {
 	{"Novell.Directory.Ldap", 0},
 	{"PEAPI", 0},
 	{"System", 0},
+	{"System.AppContext", 0},
```

Comment:
@marek-safar if we don't add the Facades/ directory there then the facades are not in any search path so it's no wonder we can't find them.

What we can do is remove the `!strcmp (runtime->framework_version, "4.5")` which constrains it only to desktop profile.